xdp-dump/tests/test-xdpdump.sh

```diff
@@ -341,7 +341,7 @@ test_multi_pkt()
     for PKT_SIZE in "${PKT_SIZES[@]}" ; do
 
         PID=$(start_background_no_stderr "$XDPDUMP -i $NS -p xdp_test_prog_with_a_long_name --rx-capture=entry,exit")
-        timeout 4 $PING6 -W 2 -s "$PKT_SIZE" -c 20000 -f "$INSIDE_IP6" || return 1
+        $PING6 -W 2 -s "$PKT_SIZE" -c 20000 -f "$INSIDE_IP6" || return 1
```

Comment:
The timeout should not be deleted, as it could cause the test to run for an extremely long time on failure.

Reply:
We could increase it to something that should never be hit in practice but still won't leave the test running forever? Like 30 seconds or so?

Reply:
Guess the question was, why was it removed? Was it causing a timeout before the test was actually completed?

If so, yes then changing it to "timeout 30 ..." would be better than not having it.

Reply:
While initially testing it, even after increasing the time, it sometimes caused a timeout. But now, I cannot reproduce that anymore, so I will just use the longer timeouts.